modules/vcf2maf/1.0/config/default.yaml

```diff
@@ -0,0 +1,23 @@
+lcr-modules:
+    
+    vcf2maf:
+        inputs:
+            # Available wildcards: {out_dir}, {seq_type}, {genome_build}, {tumour_id}, {normal_id}, {pair_status}, {vcf_name}
+            vep_cache: "reference/ve"p_caches/"
+
+        options:
+            vcf2maf: "--filter-vcf 0 --vcf-tumor-id TUMOR --vcf-normal-id NORMAL --cache-version 97"
+            # --filter-vcf     A VCF for FILTER tag common_variant. Set to 0 to disable [~/.vep/ExAC_nonTCGA.r0.3.1.sites.vep.vcf.gz]
+            #--species        Ensembl-friendly name of species (e.g. mus_musculus for mouse) [homo_sapiens]
+            #--cache-version  Version of offline cache to use with VEP (e.g. 75, 84, 91) [Default: Installed version]
+            species: "homo_sapiens"
+            vep_genome_build: "97_GRCh37"
```

Comment:
Is this hard-coded to only support one of the genome builds?

Reply:
I added that for auto-installation because 86-GRCh37  wasn't in the gsc link. For vcf2maf-1.0 I think I'll ask the users to provide a path to vep caches. I will keep working on the auto installation for the next versions